src/server/server-html.jsx

```diff
@@ -251,6 +251,8 @@ export default function ServerHTML({
 
                                 googletag.cmd.push(function() {
                                   googletag.pubads().disableInitialLoad();
+                                  googletag.pubads().setTargeting("edition", ["new-york"]);
```

Comment:
FWIW, this was included with the previous code. I removed it when I was troubleshooting the prebid code.